src/js/utils/to-lower-case.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @file to-lower-case.js
```

Comment:
This could potentially be combined with the to-title-case file into a string-cases.js file or something.
We probably want a test for this as well, a la https://github.com/videojs/video.js/blob/master/test/unit/utils/to-title-case.test.js